UT4MasterServer/Services/StatisticsService.cs

```diff
@@ -509,11 +509,12 @@ public async Task<long> DeleteOldStatisticsAsync(int days = 30, bool skipFlagged
 
 		var removeBeforeDate = DateTime.UtcNow.Date.AddDays(-days);
 
-		var filter = Builders<Statistic>.Filter.Lt(f => f.CreatedAt, removeBeforeDate);
-		
+		var filter = Builders<Statistic>.Filter.Lt(f => f.CreatedAt, removeBeforeDate) &
+					 Builders<Statistic>.Filter.In(f => f.Window, new List<StatisticWindow>() { StatisticWindow.Daily });
```

Comment:
use `Eq` instead of `In` if just testing if its one specific value

Reply:
That is left because I will remove both Daily and DailyMerged statistics, but this PR doesn't have DailyMerged enum member yet.